### DISCUSSION TITLE
Streampark uses summaries

### DISCUSSION THREAD

**Discussion by fi-ecavc** [apache/streampark#3207] [General]
After thinking about it, I didn't want to summarize it, because I mentioned it and I probably won't reply, because the response rate of this project group is not high, but since I used this, let's put it forward:
1, pod template, as long as serviceAccount is added, the error is made, and the thing is finally removed. I always feel that some parameters in the pod template do not seem to take effect, and it is easy to make mistakes due to formatting. It is recommended that the project team provide a comprehensive, standard template.
2. As long as the base image is updated, the cluster is always prompted when running the task again, but the original cluster has actually been deleted, and a new application can only be copied.
3, the whole development process formed zombie or orphan pods too many, all kinds of CrashLoopBackOff, Init: Error, ImagePullBackOff, and did not provide a cleaning method, some jobs were deleted, or cleaned up, k8s resource-related pods should also have a special cleanup mechanism.
4. The logs are too scattered, and you must go to at least 3 places to see the logs to troubleshoot the problem
5. Normally executed tasks, there is no log prompt after a period of failure, and the related pods are no longer visible, it is difficult to troubleshoot the cause.
6. It is recommended to see the conf configuration of each task and the related process cm on the page in order to troubleshoot and solve the problem.
7. How to initialize related applications, such as initializing Paimon's catalog, etc., it is recommended that the project team provide guidance and instructions.
8, there are some other details of the problem, may be some in the expert is not a problem, but also suggest that the project team can seriously answer, after all, the user industry has specialization, the level is not the same, otherwise you can only spend a lot of time constantly trying.
In general, it is recommended that the official documentation be a little more detailed and accurate.

**Comment by caicancai** [comment]
Thank you for your opinion, we will refer to and draw on it, the official document we are currently improving, it is estimated that it will come out with 2.2.0. Thank you very much for your comments
